src/sdk/formatting/RelativeDate.ts

```diff
@@ -1,5 +1,5 @@
-const startOf = (date: Date, unit: 'day' | 'hour' | 'minute' | 'second'): Date => {
-	date = new Date(date);
+const startOf = (date: Date, unit: 'day' | 'hour' | 'minute' | 'second', convertToLocal?: boolean): Date => {
+	date = convertToLocal ? new Date(date.getTime() - (date.getTimezoneOffset() * minute)) : new Date(date);
```

Comment:
How was this causing SF-3016?

Reply:
Feels like there should be some test coverage around this.

Reply:
I switched all the date/time tests to use Phoenix MST instead of UTC, and added a few new tests to calculate days using timestamps immediately before & after midnight.
Discovered some additional issues after doing that, and made some more changes.
All are passing now.

Reply:
As soon as I said that... looks like AM/PM calc is failing in CI. It's working on my local. Taking a look at that

Reply:
Fixed AM/PM